### DISCUSSION TITLE
[Bug]  I'm Having problems installing scoop

### DISCUSSION THREAD

**Discussion by jjjjn2** [ScoopInstaller/Scoop#5108] [General]
I followed the instruction from scoop.sh and set the execution policy as the instructions said and I got this error:

![Screenshot 2022-08-13 205027](https://user-images.githubusercontent.com/65252351/184518014-175c999b-1f0c-428a-90e9-28a3c9f193db.png)

**Comment by rashil2000** [comment]
Can you delete the ~\scoop folder and try again?

**Reply by jjjjn2** [reply]
I wasn't able to find the folder

**Reply by rashil2000** [reply]
What is the output of `Get-ChildItem $env:UserProfile -Force`?

**Reply by jjjjn2** [reply]
'Get-ChildItem' is not recognized as an internal or external command,
operable program or batch file.

**Comment by rashil2000** [comment]
Run it in PowerShell

**Reply by jjjjn2** [reply]
I did, that's the error I got

**Reply by rashil2000** [reply]
No, you ran it in Command Prompt (CMD). That's a CMD error.

**Reply by jjjjn2** [reply]
Ok, I ran it in PowerShell and it is the directories of my user

**Reply by rashil2000** [reply]
Paste the output here

**Reply by jjjjn2** [reply]
![Screenshot 2022-09-04 235425](https://user-images.githubusercontent.com/65252351/188338438-bcc8e5fe-7dbd-4a2a-afc0-e2a8a6e0d3b1.png)

**Reply by rashil2000** [reply]
Can you try installing as admin https://github.com/ScoopInstaller/Install#for-admin?

**Reply by jjjjn2** [reply]
That worked perfectly, thank you so much
